core/azure-core/src/main/java/com/azure/core/util/logging/ClientLogger.java

```diff
@@ -65,9 +65,6 @@
     private static final int DISABLED_LEVEL = 5;
 
     private static final int DEFAULT_LOG_LEVEL = DISABLED_LEVEL;
```

Comment:
Given the changes to logging is this still needed? At this time all this does is defaults the log method to a no-op call.

Reply:
Yeah, I agree. This is not required anymore. I also had a question about `TRACE_LEVEL`. There's no method that allows users of `ClientLogger` to log at `TRACE_LEVEL`. Should we remove that level or add a method to support logging at `TRACE_LEVEL`?

Reply:
I removed trace level as it seemed duplicative to have tracing logging and actual tracing

Reply:
Ok, then I'll remove `TRACE_LEVEL`.